specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-09-02-preview/examples/ManagedClustersCreate_EnableCustomCATrust.json

```diff
@@ -60,6 +60,11 @@
           "clientId": "clientid",
           "secret": "secret"
         },
+        "securityProfile": {
+          "customCATrustCertificates": [
+            "base64EncodedCert"
```

Comment:
Wasn't sure if we need an actual base64 encoded cert here as an example?

Reply:
I would use a dummy base64 value maybe? That way it looks like it would really look on the wire.